Coursework/2-Functions/exercise1.js

```diff
@@ -8,11 +8,11 @@ please fix this bug so we see the result is "360"
 */
 
 function calculateAgeInMonths(ageInYears){
-    let ageInMonths = ageInYears * 12;
+    return ageInMonths = ageInYears * 12;
 
```

Comment:
Excellent. If you wanted you could just have `return ageInYears * 12;` since you are then reading it from the function return - you don't have to store it in a variable before returning it.